### ISSUE TITLE
ARIA attributes "aria-label" are not valid for the element <body> with implicit ARIA role "generic" violates WCAG 2.1 SC 4.1.2 Name, Role, Value (Level A)

### ISSUE THREAD

**Issue by RushikeshTrivedi** [microsoft/vscode#316415]
Does this issue occur when all extensions are disabled?: Yes/No

- VS Code Version: 1.119
- OS Version: Windows 11

Steps to Reproduce:
1. Run Accessibility test on Source control view

Issue:
Our Accessibility tests are failing for The ARIA attributes "aria-label" are not valid for the element <body> with implicit ARIA role "generic". We are using built-in or default use VSCODE views without any customization. Similar views in VSCODE like Source Control is also failing for same reason.

For all user interface components (including but not limited to: form elements, links, and components generated by scripts), the name and role can be programmatically determined; states, properties, and values that can be set by the user can be programmatically set; and notification of changes to these items is available to user agents, including assistive technologies.

Rationale
This requirement ensures that Assistive Technologies (AT) can gather information about, activate, set, and receive status of all user interface components.

Note: This requirement is primarily for developers who develop or script their own custom user interface components. Standard HTML controls already meet this requirement when used according to specification.

For more information: https://www.w3.org/WAI/WCAG22/Understanding/name-role-value

Below is the screen shot of the VSCODE views and our views impacted:

<img width="3571" height="1581" alt="Image" src="https://github.com/user-attachments/assets/3f96daa3-8754-4179-b5f3-7b4ef4df009a" />

<img width="3628" height="1274" alt="Image" src="https://github.com/user-attachments/assets/d690ee13-8553-418e-a162-4fe912daa65a" />

<img width="3759" height="818" alt="Image" src="https://github.com/user-attachments/assets/d99d526e-ca20-4ec7-963c-a5d917a6f895" />

Custom interactive user interface components often do not expose proper role, state, and property information needed for assistive technologies. As a result, people with disabilities have difficulty identifying and interacting with the components. Common examples are custom menus, trees, data grids, accordions, and dialog boxes. Ensuring proper markup and keyboard operation enables people with disabilities to identify and interact with the custom components in the same manner they identify and interact with standard components.

**Comment by RushikeshTrivedi** [comment]
With v1.126: Issue still exists

<img width="3088" height="2140" alt="Image" src="https://github.com/user-attachments/assets/bbf9aa30-7ace-45e4-9759-e473816e7292" />

<img width="3121" height="2050" alt="Image" src="https://github.com/user-attachments/assets/ee8eab5b-6baa-405f-947d-fd85366dd285" />

<img width="3158" height="1822" alt="Image" src="https://github.com/user-attachments/assets/d7358914-8689-4439-92ee-ed9a5adaf7c7" />
